app/src/main/kotlin/com/wire/android/migration/feature/IsScalaMigrationPossibleUseCase.kt

```diff
@@ -0,0 +1,10 @@
+package com.wire.android.migration.feature
+
+import android.content.Context
+import com.wire.android.migration.util.ScalaDBNameProvider
+
+class IsScalaMigrationPossibleUseCase(
```

Comment:
Is this use case necessary since we already have `shouldMigrate(): Boolean` in `MigrationManager`? 🤔